### DISCUSSION TITLE
Dementonite tools are uncraftable yet show in fabricator menu

### DISCUSSION THREAD

**Discussion by AhpiTai** [FakeFishGames/Barotrauma#10754] [Unstable feedback]
Despite the talent that allows Mechanics to craft dementonite tools being removed, they still show up as options in the fabricator menu, yet are uncraftable. It still labels them as requiring a talent for the player to craft them, thus making their appearance in the fabricator menu redundant.

**Comment by mygamingaccount** [comment]
Is this the first time a bug report gets posted in Discussions and not the other way?
